### ISSUE TITLE
What is the weight_decay value used?

### ISSUE THREAD

**Issue by look4pritam** [LynnHo/AttGAN-Tensorflow#38]
Default value for weight_decay is 0.0.
What is the weight_decay value used for training the model?

**Comment by look4pritam** [comment]
What are the hyper parameters used for training the model?

**Comment by LynnHo** [comment]
@look4pritam It's just zero as you see, i.e., no weight_decay is applied to the model.

**Comment by look4pritam** [comment]
I was trying to reproduce your results using TensorFlow-2 version. Here generator loss is decreasing from 70. to 7. but discriminator loss is oscillating around 10.41. I am able to recreate input image with good quality, modified attributes are not getting applied. What may be the error?

**Comment by look4pritam** [comment]
You can find implementation at https://github.com/look4pritam/TensorFlowExamples/blob/master/GAN/AttGAN/AttGAN-Attributes.ipynb

**Comment by LynnHo** [comment]
@look4pritam I think you should print the losses separately, and find out which loss is abnormal so that you can check the corresponding part.
![image](https://user-images.githubusercontent.com/16457298/82676172-846d8500-9c78-11ea-9992-2aca4a6619c1.png)
![image](https://user-images.githubusercontent.com/16457298/82676187-8afbfc80-9c78-11ea-8cc1-13586650f37e.png)

**Comment by look4pritam** [comment]
Thanks for prompt reply.

Are 60 epochs sufficient?
Reconstructed image is good.
Discriminator output is bad.
I think more epochs are needed.
But as suggested by you, I will check each losses to fix the problem.

Thanks and Regards

**Comment by LynnHo** [comment]
@look4pritam 60 epochs should work. Actually, you can see the editing effect in about 10 epochs.

**Comment by look4pritam** [comment]
Thanks a lot, it will save time.
I can train for epochs just more than 10 and see for attribute changing.

Before that, I wiil find out problematic loss part as per your suggestion.

Thanks a lot for your help.

**Comment by look4pritam** [comment]
Thank you very much for your reply. Now I am able to train the model.
